gettsim/pensions.py

```diff
@@ -90,7 +92,7 @@ def _regelaltersgrenze(person):
 
 def _rentenwert_until_2017(tb_pens, yr):
```

Comment:
Isn't a single function for rentenwert better than two functions to centralize the logic?